napari/plugins/__init__.py

```diff
@@ -1,18 +1,19 @@
 import sys
 
-from napari_plugins import napari_hook_implementation
-
-from .exceptions import PluginError, PluginImportError, PluginRegistrationError
-from .manager import PluginManager
+from napari_plugin_engine import PluginManager
+from . import hook_specifications
+from . import _builtins
 
 # the main plugin manager instance for the `napari` plugin namespace.
-plugin_manager = PluginManager()
+plugin_manager = PluginManager(
+    'napari', discover_entry_point='napari.plugin', discover_prefix='napari_'
```

Comment:
out of curiosity, can entrypoint prefixes use `-`?

Reply:
good question!  I need to look into to see what would happen if a package was named something like `napari-image` but its top level module was named `naimage` :)